zcu/encryption.py

```diff
@@ -20,27 +28,37 @@ def aes_decrypt(cipher, aes_key):
     encrypted_data = BytesIO()
     while True:
         aes_chunk = struct.unpack('>3I', cipher.read(12))
-        encrypted_data.write(cipher.read(aes_chunk[0]))
+        # aes_chunk[0] -> length not padded
+        # aes_chunk[1] -> padded length
+        if digi:
+          chunk_size=aes_chunk[1]
+        else:
+          chunk_size=aes_chunk[0]
+        encrypted_data.write(cipher.read(chunk_size))
         if aes_chunk[2] == 0:
             break
     encrypted_data.seek(0)
-    aes_cipher = AES.new(aes_key, AES.MODE_ECB)
+    if digi:
```

Comment:
can you use longer names rather than `k` and `v` otherwise the linter complains...

Reply:
changed to key and iv (or is iv still to short? it's what the aes literature uses)